src/components/Bookings/Bookings.jsx

```diff
@@ -1,11 +1,24 @@
-import React, { useState } from "react";
+import React, { useState, useEffect } from "react";
 import Search from "../Search/Search";
 import FakeBookings from "../../data/fakeBookings.json";
 import SearchResults from "../SearchResults/SearchResults";
 import AddBookingForm from "../AddBookingForm/AddBookingForm";
 
 const Bookings = () => {
-  const [bookings, setBookings] = useState(FakeBookings);
+  const [bookings, setBookings] = useState([]);
+
+  useEffect(() => {
+    fetchBookings();
+  }, []);
+
+  const fetchBookings = () => {
+    try {
+      // Replace with actual API call
+      setBookings(FakeBookings);
+    } catch (error) {
+      console.error("Error fetching bookings:", error.message);
```

Comment:
This is a good error message, but the average user will never see it - users don't look in the console. You can put more detail in the console but error messages should always be obvious to the user